### ISSUE TITLE
Bug report : cannot find localsend in vm

### ISSUE THREAD

**Issue by newyorkthink** [localsend/localsend#927]
### Describe the bug

![image](https://github.com/localsend/localsend/assets/87791643/b37c8838-5c06-4ea5-ae96-bb6beba7b8e3)
![image](https://github.com/localsend/localsend/assets/87791643/a1deac0a-1313-4087-9154-8aef43ef405c)

### To reproduce

2. Click on '...'
3. Scroll down to '...'

### Expected behavior

cannot find localsend

**Comment by MMachado05** [comment]
I'm not a LocalSend dev, but this Issue looks very lacking in detail. You have two screenshots with arrows and a three-sentence problem description, but no steps to reproduce, no system or localsend version information... I think this would be a lot easier to solve if there were more details regarding the conditions under which this problem arises!

**Comment by MMachado05** [comment]
Yes, this overall seems like very poor Issue submission practice. @newyorkthink, would you be willing to close both this issue and #906 and recreate it as a more detailed feature request ***or*** bug report, exclusively? That way the repo will remain clean and there's a better likelihood a dev will be able to implement what you are proposing.

**Comment by Joackk** [comment]
I also encountered this problem when localsend was used in the virtual machine and he could not detect localsend on the physical machine. However, he can detect localsend in other virtual machines. I run the virtual machine of wind7 in the desktop computer, and then enable localsend, and at the same time enable localsend on the desktop, but the two can not detect each other, the ip address can communicate with each other, but manual addition can not find. Then I started a wind10 virtual machine in my laptop, and strangely enough, localsend on two different physical machines could detect each other. But none of them could detect the physical machine.

**Comment by Joackk** [comment]
Steps to Reproduce:

Run localsend on a physical machine.
Run localsend on a virtual machine within the same network.
Attempt to detect the virtual machine's localsend instance from the physical machine.
Expected Result:
The localsend instance on the physical machine should detect the instance running on the virtual machine.

Actual Result:
The localsend instance on the physical machine does not detect the virtual machine.  However, the IP addresses of the machines are reachable via ping, indicating network connectivity.

Additional Information:

The firewall settings on both the physical and virtual machines have been configured to allow traffic for localsend.
Testing with another physical machine (a laptop) showed that localsend instances on physical machines could detect each other, but neither could detect an instance running on a virtual machine.
Running a virtual machine on the laptop and executing localsend there showed that virtual machines could detect each other but not physical machines.
This issue seems to indicate a potential problem with localsend in recognizing instances running in virtual environments, despite active network connections and proper firewall configurations.

**Comment by Joackk** [comment]
@Tienisto Has this issue been noticed? If not, I would like to raise this question again and describe the reproduction process of the galaxy

**Comment by newyorkthink** [comment]
> @Tienisto Has this issue been noticed? If not, I would like to raise this question again and describe the reproduction process of the galaxy 

yes ,you need re-send issus, the author cannot fixed it
